tools/interop_matrix/client_matrix.py

```diff
@@ -56,7 +56,7 @@ def should_build_docker_interop_image_from_release_tag(lang):
     'cxx': ['cxx'],  # This is actually debian8.
     'go': ['go1.8', 'go1.11'],
     'java': ['java'],
-    'python': ['python'],
+    'python': ['python', 'pythonasyncio'],
```

Comment:
❤️

Reply:
just out of curiosity, the matrix is built by crossing all of the language implementations, so, for example, ending up by having an interop test between `java` and `pythonasyncio`? Interop test are executed as part of the Kokoro CI?

Reply:
That's right. And we should also be creating frozen images for each release of gRPC in order to test backwards compatibility.

Reply:
Yes. Not only between languages, but also between historical versions. https://github.com/grpc/grpc/blob/master/tools/interop_matrix/client_matrix.py#L77

Yes, interop tests are executed for every push. It's named `Interop Cloud-to-Cloud Tests`.